packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -7,6 +7,13 @@ const propTypes = {
    * The source URL of the content to load.
    */
   src: PropTypes.string.isRequired,
+  /**
+   * The title of the content in the frame.
+   *
+   * ![IMPORTANT](https://badgen.net/badge/UX/Accessibility/blue) It is critical to screen reader users that the
+   * title of the frame is set to a meaningful title for the content inside the frame.
+   */
+  title: PropTypes.string,
```

Comment:
I think this should be rename to `titleText` OR `iframeTitle` because in the past we had renamed prop `title` with `text` for terra header components as it would conflict with HTML `title` attribute.

Reply:
@supreethmr this is the HTML title attribute that I am asking to be added. Without it, iframes will not pass accessibility requirements.

Reply:
@supreethmr - this prop is for the EmbeddedContentConsumer component which will be passed down to the actual iframe attribute. I don't see any conflict with it.